### ISSUE TITLE
Question : Is there a way to show messages?

### ISSUE THREAD

**Issue by AquilaRapax** [anteo/redmine_custom_workflows#202]
Hi, i just stumbled over this amazing plugin and installed it on a test system. I started to play around with it and wondered, if there is the possibility to render some text, e.g. a information box or even some output into the browsers console.

**Comment by AirTibu** [comment]
Hi, 

Yes, I agree, this plugin is the best in the world of Redmine. I started using it at a large company where I don’t have access to source code, so the possibility of customization  by myself is minimized.

This plugin opened every door and gave me limitless possibilities to use it.

With some creativity we can use it for amazing things, users love it, almost every week there is a request from a department to handle their processes in this system as well. We use it for practically everything, but not for what it is designed for: project management :D

I met with Redmine 1 year ago without any ruby ​​and IT knowledge. By now I had managed to gather so much basic ruby ​​knowledge that I could solve all what I want, except two of the problems: 

- sending a flash notification
- setting up scheduled tasks. 

Obviously there are several plugins for the latter, but flash notification would be a really gap-filling enhancement.

This story is not new, check this issue: https://github.com/anteo/redmine_custom_workflows/issues/39

Maybe this feature will be available in the next version. Fingers crossed :)
